packages/build/src/plugins/child/mutations.js

```diff
@@ -44,10 +43,28 @@ const preventObjectMutations = function (value, keys, { state, configMutations,
   return value
 }
 
+// Inverse of `preventObjectMutations()`
+const removeProxies = function (value) {
```

Comment:
This function won't be needed anymore once the logic is moved to the parent process.